src/cirrus/lib/events.py

```diff
@@ -185,7 +185,7 @@ def started_processing(
     ):
         if isotimestamp is None:
             isotimestamp = self.isotimestamp_now()
-        self.statedb.set_processing(payload_id, execution_arn, isotimestamp)
+        self.statedb.set_processing(payload_id, isotimestamp)
```

Comment:
This is because CLAIMED is setting the execution arn and we don't need to any longer for this state update, right? If so feel free to resolve, I'm just checking my understanding.